Simplified/Reader2/NYPLReaderBookmarksBusinessLogic.swift

```diff
@@ -337,7 +345,76 @@ class NYPLReaderBookmarksBusinessLogic: NSObject, NYPLReadiumViewSyncManagerDele
     completion()
   }
 
-  // TODO: Sync reading position
+  // MARK: - Read Position
+  
+  /// Post the read position to server if we have synced the read position from server
+  /// There is a 120 seconds interval to avoid high frequency of requests
+  /// - Parameters:
+  /// - locationString: A json string that contains required information to create a Locator object
```

Comment:
We are storing the location info as a string in NYPLBookLocation, so any format would work as long as we know how to decode it. I am now using dictionary instead of string, and generating a json string from the dictionary.